### DISCUSSION TITLE
Integration of Great Expectations with Robot Framework

### DISCUSSION THREAD

**Discussion by nehadahiya88** [great-expectations/great_expectations#4180] [Request For Comment]
**Is your feature request related to a problem? Please describe.**
I want to use the Great Expectations with Robot Framework.

**Describe the solution you'd like**
Library compatible with robot framework

**Describe alternatives you've considered**
exploring

**Additional context**

**Comment by NathanFarmer** [comment]
Hi @nehadahiya88! Thanks for raising this idea. It isn't on our immediate roadmap, but if you would like to make a contribution and collaborate with the community, we would be happy to offer our guidance.

**Reply by yingqiaozheng** [reply]
Hi @NathanFarmer , do we have any update on this work so far? We  are trying to integrate great expectations into Robot framework which using by our company. so that we can use the  macros/ libraries already exist in great expectations. Thank you.

**Reply by NathanFarmer** [reply]
Thanks for reaching out @yingqiaozheng. We do not have any immediate plans to build this integration. Can you elaborate on how you plan to use GX with Robot Framework? There might be a workaround in the meantime.
